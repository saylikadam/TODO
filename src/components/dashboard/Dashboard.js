import React from 'react';
import './Dashboard.scss';

export default class Dashboard extends React.Component {
    render() {
        return (
            <div className="main-container">
                Hello world
            </div>
        );
    }
}