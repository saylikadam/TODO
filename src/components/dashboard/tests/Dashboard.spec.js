import React from 'react';
import { render } from 'react-testing-library';
import Dashboard from '../Dashboard';

describe('<Home />', () => {
    let result;
    beforeEach(() => {
        result = render(<Dashboard/>);
    });

    it('should render Home page', () => {
        const { container } = result;
        expect(container).toMatchSnapshot();
    });


});