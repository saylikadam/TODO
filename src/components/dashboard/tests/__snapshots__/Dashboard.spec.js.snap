// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render Home page 1`] = `
<div>
  <div
    class="main-container"
  >
    Hello world
  </div>
</div>
`;
